src/lib/api/helper/interceptors.ts

```diff
@@ -105,6 +106,31 @@ function interceptorFn(options) {
     };
 }
 
+function responseInterceptorFn(response) {
```

Comment:
export single response interceptor function and use it as a callback function @Shivraj0